restlight-core/src/main/java/io/esastack/restlight/server/handler/ChannelWrapper.java

```diff
@@ -0,0 +1,61 @@
+/*
+ *
+ * Copyright 2022 OPPO ESA Stack Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.esastack.restlight.server.handler;
+
+import esa.commons.Checks;
+import io.netty.channel.Channel;
+
+/**
+ * the ChannelWrapper mainly shield the detail use of {@link io.netty.channel.Channel}.
+ *
+ * @author chenglu
+ */
+public final class ChannelWrapper {
```

Comment:
I think what we need is an interface. The goal is not to rely on netty's channel, and the underlying dependencies can be changed at any time. The interface is more suitable for our needs.